WordPress/src/main/res/layout/add_content_bottom_sheet.xml

```diff
@@ -13,40 +17,5 @@
         android:clipToPadding="false"
         android:descendantFocusability="beforeDescendants"
         android:scrollbars="vertical"
-        android:layout_marginTop="@dimen/margin_extra_large"
```

Comment:
Hi @develric, thanks for the review!

I made some experimentations on that front and decided to try to get as closer as possible to the specs on `pauD4L-B3-p2#comment-523`, without having to change the `RecyclerView` item layout from the **My Site** bottom sheet – since its header is an item itself and it already has some padding.

The top margin is a tiny bit different from the **Reader** bottom sheet, but not by much: less than `8dp` from what I can see. Keeping the `16dp` top margin would make that difference much more noticeable.

Here's some side-by-side comparisons:

| My Site bottom sheet | Reader bottom sheet |
|---|---|
| ![my-site](https://user-images.githubusercontent.com/830056/74791387-14c2b500-5299-11ea-8e23-6fad458a5c22.jpeg) | ![reader](https://user-images.githubusercontent.com/830056/74791395-1ab89600-5299-11ea-91bf-975c03b69bda.jpeg) |

And here's the same comparison, but with a gridline and layout bounds:

| My Site bottom sheet with grid | Reader bottom sheet with grid |
|---|---|
| ![my-site-grid](https://user-images.githubusercontent.com/830056/74791388-168c7880-5299-11ea-96bc-aec93aec0c8b.jpeg) | ![reader-grid](https://user-images.githubusercontent.com/830056/74791398-1ab89600-5299-11ea-80d2-05df319965a4.jpeg) |

Reply:
Hey thanks for the ping @develric ! 

When I was updating these styles and adding the headers I worked on them together. So the spacing between the heading (Add New / Following) and the drag handle should be the same in both. 

> FWIW the original design at the time here was to have all the elements equispaced by 32dp top and bottom of the bottom sheet included

I think this direction might have been before we had the drag handle, or maybe even the heading. So I am not sure if it holds true. As mentioned by @renanferrari pauD4L-B3-p2#comment-523 would be the latest specs. I didn't have one there for the create sheet as we weren't working on it at that point. So here's a side by side with specs

![specsx2](https://user-images.githubusercontent.com/50576199/74936871-d9340200-53e2-11ea-9bcf-30b125a76479.jpg)


In my mocks I had 16dp in both. But depending how the padding is defined e.g. 'since its header is an item itself and it already has some padding.' it can be hard to say precisely where we should add the padding and how much.

Looking at it now it seems like the reader one has a bit too much. Speaking of balancing if you look at the create sheet - I have 24dp at the bottom below the content. Then I have 26dp at the top in total (16dp + 5dp drag handle + 5dp above the drag handle). So they are almost equal. The drag handle is a bit unusual. Because it is not content I believe it almost appears the same as white space, and optically looks like part of the padding. Hope that makes sense.

All that being said this PR looks correct to me. If anything I would tweak the spacing between "Following" and the drag handle in the reader to match what we have here.

Questions:
- should we standardise that space and have it as part of the sheet component? i.e. have some padding as part of the sheet header to make all sheets consistent? I can't fully understand the gridlines and bounds image above to understand the breakdown of where the spacing exists right now. Does the 'Following' heading have some padding built in?
- I mentioned the drag handle is 5dp and 5dp padding above. I found this was a common dimension in other apps. But looking at the grid lines reminds me that 4dp and 4dp might closer align to standard Android. Do you think we should change it?

Reply:
@osullivanchris Thank you for reviewing this!

> If anything I would tweak the spacing between "Following" and the drag handle in the reader to match what we have here.

That's definitely something we can do. I will tweak that and get back to you in a few.

> should we standardise that space and have it as part of the sheet component? i.e. have some padding as part of the sheet header to make all sheets consistent?

Right now, I don't think we should, as the content in these two bottom sheets are implemented quite differently and they dictate everything that goes below the handle, including that space. If we wanted to change that, we would need to change some implementation details on the create bottom sheet that I think is out of scope for this PR. However, that's definitely something we could do later, as part of our efforts to create a shared bottom sheet component.

> I mentioned the drag handle is 5dp and 5dp padding above. I found this was a common dimension in other apps. But looking at the grid lines reminds me that 4dp and 4dp might closer align to standard Android. Do you think we should change it?

Yes! I'm glad you mentioned this. It was something I thought about suggesting, but eventually forgot to. Good call 👍

Reply:
> That's definitely something we can do. I will tweak that and get back to you in a few.

Ok cool. Just co-ordinate with @develric as he wants to release that stuff soon. And its not a blocker, if there are any issues.

> However, that's definitely something we could do later, as part of our efforts to create a shared bottom sheet component.

sounds good. We should do a share component properly later on, rather than hacking at the separate implementations now at a late stage.

> Yes! I'm glad you mentioned this. It was something I thought about suggesting, but eventually forgot to. Good call 👍

Cool. If that's an easy fix, I am happy to swap to 4 and 4. As long as it is done for both sheets :) (If we had the component you would only have to fix it once... - Just kidding!)

Reply:
Here are some updated screenshots:

| My Site bottom sheet | Reader bottom sheet |
|---|---|
| ![my-site](https://user-images.githubusercontent.com/830056/74969983-56b83c00-53fc-11ea-839a-5f1577a49897.jpeg) | ![reader](https://user-images.githubusercontent.com/830056/74969973-54ee7880-53fc-11ea-8741-783c69c48d34.jpeg) |

And here's the same comparison, but with a gridline and layout bounds:

| My Site bottom sheet with grid | Reader bottom sheet with grid |
|---|---|
| ![my-site-grid](https://user-images.githubusercontent.com/830056/74969985-5750d280-53fc-11ea-872e-9fdc61beb1b2.jpeg) | ![reader-grid](https://user-images.githubusercontent.com/830056/74969988-57e96900-53fc-11ea-9aac-49a97dae60de.jpeg) |

> (If we had the component you would only have to fix it once... - Just kidding!)

Ah, but we _do have_ a shared component for the handle, so I only had to change it once 😉– 
what we are currently lacking is a shared component for the _whole_ bottom sheet. For things like the handle, the title and the borders, we already have some shared components and resources (styles, dimensions).

Reply:
Sounds great, and looks great! Thank you @renanferrari !